src/modules/10-common/components/MultiTypeFieldSelector/MultiTypeFieldSelector.tsx

```diff
@@ -43,7 +43,7 @@ export interface MultiTypeFieldSelectorProps extends Omit<IFormGroupProps, 'labe
 }
 
 export interface ConnectedMultiTypeFieldSelectorProps extends MultiTypeFieldSelectorProps {
-  formik: FormikContext<any>
+  formik: FormikContextType<any>
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
  formik: FormikContextType<unknown>
```